### PR TITLE
Issue #574 Updated MAINTAINERS list

### PR DESCRIPTION
**Pull request by budhrg**
Let me know if there is any incorrect information here.

@hferentschik @gbraad @LalatenduMohanty @praveenkumar @thatdocslady @Preeticp

### CODE REVIEW

**Comment by LalatenduMohanty** [comment]
I do not see any significant advantage off having a maintainer file. I think @hferentschik and @gbraad agree to it.

**Comment by budhrg** [comment]
Hi @jimmidyson ,

We have also been discussing around removing the file. What is your opinion on it?

File: MAINTAINERS [hunk 1]
```diff
@@ -1 +1,10 @@
-Jimmi Dyson <jimmidyson@gmail.com> (@jimmidyson)
+Jimmi Dyson           <jimmidyson@gmail.com>        ( @jimmidyson )
+Hardy Ferentschik                                   ( @hferentschik )
```
Comment (gbraad) [inline MAINTAINERS]: hardy's email would still be available from commits, so please add

Comment (budhrg) [inline MAINTAINERS]: @gbraad :smile: , he only told me to remove it otherwise it was there previously :)

**Comment by budhrg** [comment]
@gbraad 
> but I thought the file would be removed... ?

Waiting for @jimmidyson opinion here.

File: MAINTAINERS (hunk 1 above)
Comment (gbraad) [inline MAINTAINERS]: are those twitter handles or freenode nicks?

Comment (gbraad) [inline MAINTAINERS]: @hferentschik really remove? people can still track it down to be

```
Author: Hardy Ferentschik <hardy@hibernate.org>
```

Aesthetically it looks weird now... ... really, Hardy? just add... ;-)

**Comment by hferentschik** [comment]
@budhrg imo we can go ahead and delete this file. Maybe you can update the issue and pull request accordingly?

**Comment by LalatenduMohanty** [comment]
@budhrg I am going to close this PR as you will send another PR for removing the file and we can add @jimmidyson as a reviewer in that PR.

**Pull request closed without merging**